.vscode/cspell.json

```diff
@@ -21,11 +21,49 @@
     "*.sha512",
     "*.ico",
     "*.pri",
+    "*.trx",
+    "*.binlog",
+    "*.log",
+    "*.sln",
+    "*.user",
     ".vscode/cspell.json"
   ],
   // cspell is not case sensitive
   // Sort words alphabetically to make this list easier to use
-  "words": ["azsdk", "pwsh"],
+  "words": [
+    "apos",
+    "aragorn",
+    "azsdk",
+    "blazor",
+    "contoso",
+    "deserializes",
+    "diagnoser",
+    "epsg",
+    "expando",
+    "finalizer",
+    "finalizers",
+    "ints",
+    "lucene",
+    "mgmt",
+    "nunit",
+    "odata",
+    "otel",
+    "overridden",
+    "parallelization",
+    "pkcs",
+    "pwsh",
+    "referer",
+    "renormalize",
+    "retriable",
+    "sauron's",
+    "struct",
+    "structs",
```

Comment:
cspell for some reason doesn't pluralize words from the dictionary.

Reply:
Pluralization is quite difficult even for a single language. Not surprising. Most don't that I've seen.